raiden/network/proxies/token_network.py

```diff
@@ -195,8 +209,17 @@ def _new_netting_channel(self, partner: typing.Address, settle_timeout: int):
         if self.channel_exists_and_not_settled(self.node_address, partner):
             raise DuplicatedChannelError('Channel with given partner address already exists')
 
+        gas_limit = self.proxy.estimate_gas(
+            'openChannel',
+            self.node_address,
+            partner,
+            settle_timeout,
+        )
+        gas_limit = get_gas_limit(gas_limit, GAS_REQUIRED_FOR_OPEN_CHANNEL)
```

Comment:
pretty nitpicky, feel free to discard:
```python
estimate = self.proxy.estimate_gas(
  ...
)
gas_limit = safe_gas_limit(estimate, GAS_REQUIRED_FOR_OPEN_CHANNEL)
```

Reply:
on the plus side for always only reusing `gas_limit` is, that this avoids accidental reuse of an older `estimate` throughout a longer scope with multiple estimations.

Reply:
I side with comment two here for now, lets fix this later.